packages/ckeditor5-table/src/utils/structure.ts

```diff
@@ -59,8 +59,17 @@ export function cropTableToDimensions(
 ): Element {
 	const { startRow, startColumn, endRow, endColumn } = cropDimensions;
 
-	// Create empty table with empty rows equal to crop height.
+	// Initialize the cropped table element.
 	const croppedTable = writer.createElement( 'table' );
+
+	// Copy table type attribute if present.
+	const sourceTableType = sourceTable.getAttribute( 'tableType' );
+
+	if ( sourceTableType ) {
+		writer.setAttribute( 'tableType', sourceTableType, croppedTable );
+	}
```

Comment:
This looks like a tight coupling. What should happen with `tableWidth` attribute?

Reply:
It handles the case when user selects part of `layout` table and then paste it directly to the editing. Without picking attributes it'll not work. `tableWidth` - dunno, while it makes to sense to copy it too.